### ISSUE TITLE
[Automation] Auto-link docs for a given operator in issues for operator onboarding

### ISSUE THREAD

**Issue by HumairAK** [operate-first/opfcli#30]
This was mentioned [here](https://github.com/operate-first/SRE/issues/216), though I'm not sure about the feasibility of this. 

I'll yield to @tumido to expand on this further.

**Comment by larsks** [comment]
@HumairAK this seems like the wrong repository for this issue: it sounds more like something that's going to live with our github automation, rather than with our cli tool.

**Comment by HumairAK** [comment]
good point, I'll transfer this to....sre I suppose. If anyone has a better suggestion let me know.
